src/resolvers/Mutation/createEvent.ts

```diff
@@ -119,40 +120,50 @@ export const createEvent: MutationResolvers["createEvent"] = async (
     );
   }
 
-  // Creates new event.
-  const createdEvent = await Event.create({
-    ...args.data,
-    creator: currentUser._id,
-    admins: [currentUser._id],
-    organization: organization._id,
-  });
+  const session = await mongoose.startSession();
```

Comment:
So your plan is to start session every time which is bad implementation. Please start one session and keep sing it for transaction throught the server

Reply:
Thank you for your review and feedback. I have taken your suggestion into consideration and implemented a single session for the entire application, now a session is initiated when the database connection is established